diff --git a/base/BUILD.gn b/base/BUILD.gn
index 08919400794cc35e0c0e41958c0b295c3fd1c2d2..2c4a46d3978ea29a20d17f53d5c7c5c564ebf592 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4617,6 +4617,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5278,6 +5279,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
