diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
index b767b3036ff57782cb69605ae09cbb64b19d7a27..3ae8fc67c5ac29e5b8f711baeb4bc0869eea7445 100644
--- a/components/content_settings/core/browser/content_settings_utils.cc
+++ b/components/content_settings/core/browser/content_settings_utils.cc
@@ -196,6 +196,7 @@ bool CanTrackLastVisit(ContentSettingsType type) {
   if (type == ContentSettingsType::PROTOCOL_HANDLERS) {
     return false;
   }
+  BRAVE_CAN_TRACK_LAST_VISIT
 
   // Chooser based content settings will not be tracked by default.
   // Only allowlisted ones should be tracked.
