diff --git a/third_party/blink/renderer/core/html/canvas/html_canvas_element.cc b/third_party/blink/renderer/core/html/canvas/html_canvas_element.cc
index 9ebe263a3596d602c80d5e50b830d34e5d5efbfc..d617e5970587be2a12186014d698626305f2d157 100644
--- a/third_party/blink/renderer/core/html/canvas/html_canvas_element.cc
+++ b/third_party/blink/renderer/core/html/canvas/html_canvas_element.cc
@@ -1391,6 +1391,7 @@ String HTMLCanvasElement::ToDataURLInternal(const String& mime_type,
     if (!data_buffer)
       return String("data:,");
 
+    BRAVE_TO_DATA_URL_INTERNAL
     String data_url = data_buffer->ToDataURL(encoding_mime_type, quality);
     base::TimeDelta elapsed_time = base::TimeTicks::Now() - start_time;
     float sqrt_pixels =
