diff --git a/BUILD.gn b/BUILD.gn
index 1414aca14ef5e1c1da56683427630212e075961b..6f0c9e4a526fbacf93324c8edf48d1e4bc9d4eee 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -862,6 +862,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3313,6 +3314,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
